File: SQL/InsertTableData/InsertTableData/Models/Station.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Text;

namespace InsertTableData.Models
{
    public class Station
    {
        public string name { get; set; }
        [DisplayName("longname.name_alias")]
        public string longname { get; set; }
        public string alpha3 { get; set; }
        public string tiploc { get; set; }
        public string dbName { get; set; }

        public override string ToString()
        {
            string column5R = dbName == "" || dbName == null ? "\\N" : dbName;
            return $"{name},{longname},{alpha3},{tiploc},{column5R}".Replace("'","''").Replace("\"","'").Replace("\\N","null");
        }

    }
}

File: src/Stations/Station.py
class Station:
  def __init__(self, arr):
    self.name = str(arr[0])
    self.longName = arr[1]
    self.alpha3 = arr[2]
    self.tiploc = arr[3]
    self.tpl = arr[4]

  def __str__(self):
    return self.name + "; " + self.longName + "; " + self.alpha3 + "; " + self.tiploc + "; " + self.tpl + ";"
